### PR TITLE
Non folding plasteel barricade

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Adds non folding plasteel barricade.

dreamseeker_S1DDU1USY9

dreamseeker_ki0T5XoVfe

Has higher hp than folding one, costs less to build than folding one (6 plasteel vs 8), but very slow to build. (Basically the same situation with metal ones, just used the same logic.

Tweakerd folding plasteel cade name to indicate that it's folding one.

# Explain why it's good for the game

Just fills a niche just as folding metal folding one did before.

Basically a port of https://github.com/Cosmic-Overlord/RU-TerraGov-Marine-Corps/pull/119

# Changelog
:cl:
add: plasteel non folding barricade
/:cl:

### CODE REVIEW

**Comment by Git-Nivrak** [comment]
What if we stopped porting rutgmc slop

**Comment by Blundir** [comment]
> What if we stopped porting rutgmc slop

Ok i remove 1 line, what changes

File: code/game/objects/structures/barricade/metal.dm [hunk 1]
```diff
@@ -279,3 +279,31 @@
...
+	maxhealth = 900
+	crusher_resistant = TRUE
+	force_level_absorption = 20
+	stack_type = /obj/item/stack/sheet/plasteel
+	debris = list(/obj/item/stack/sheet/plasteel)
+	destroyed_stack_amount = 3
+	barricade_type = "new_plasteel"
+	repair_materials = list("plasteel" = 0.45)
+
+/obj/structure/barricade/metal/plasteel/wired/New()
```
Comment (Zonespace27) [inline code/game/objects/structures/barricade/metal.dm:297]: we don't use New, also call parent before your code in Initialize and New

Comment (Blundir) [reply code/game/objects/structures/barricade/metal.dm:297]: Honestly i can just scrap that chunk entirely, it was optional

**Review by kiVts — APPROVED** [review]
Code, Dunno about balance though

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -96,7 +96,9 @@ GLOBAL_LIST_INIT_TYPED(metal_recipes, /datum/stack_recipe, list ( \
  * Plasteel
  */
 GLOBAL_LIST_INIT_TYPED(plasteel_recipes, /datum/stack_recipe, list ( \
-	new/datum/stack_recipe("plasteel barricade", /obj/structure/barricade/plasteel, 8, time = 4 SECONDS, one_per_turf = ONE_TYPE_PER_TURF, on_floor = 1, skill_req = SKILL_CONSTRUCTION, skill_lvl = SKILL_CONSTRUCTION_ENGI, min_time = 2 SECONDS),
+	new/datum/stack_recipe("folding plasteel barricade", /obj/structure/barricade/plasteel, 8, time = 4 SECONDS, one_per_turf = ONE_TYPE_PER_TURF, on_floor = 1, skill_req = SKILL_CONSTRUCTION, skill_lvl = SKILL_CONSTRUCTION_ENGI, min_time = 2 SECONDS),
+	null, \
```
Comment (Drulikar) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:100]:
```suggestion
```
image

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 3]
```diff
@@ -96,7 +96,9 @@ GLOBAL_LIST_INIT_TYPED(metal_recipes, /datum/stack_recipe, list ( \
...
+	new/datum/stack_recipe("plasteel barricade", /obj/structure/barricade/metal/plasteel, 6, time = 10 SECONDS, one_per_turf = ONE_TYPE_PER_BORDER, on_floor = 1, skill_req = SKILL_CONSTRUCTION, skill_lvl = SKILL_CONSTRUCTION_ENGI, min_time = 2 SECONDS),
```
Comment (Drulikar) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:101]:
```suggestion
	new/datum/stack_recipe("plasteel barricade", /obj/structure/barricade/metal/plasteel, 6, time = 8 SECONDS, one_per_turf = ONE_TYPE_PER_BORDER, on_floor = 1, skill_req = SKILL_CONSTRUCTION, skill_lvl = SKILL_CONSTRUCTION_ENGI, min_time = 2 SECONDS),
```
Or maybe less

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Drulikar — APPROVED** [review]

**Review by realforest2001 — APPROVED** [review]
